src/bus_constants/28.json

```
@@ -0,0 +1,166 @@
+            },
+            {
+                "stop_name": "Morton Street @ Blue Hill Avenue",
+                "branches": null,
+                "station": "mortn",
+                "order": 3,
+                "stops": {
+                    "1": [
+                        "28-1-1731"
+                    ],
+                    "0": [
+                        "28-0-11712"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Blue Hill Avenue @ Talbot Avenue",
+                "branches": null,
+                "station": "bltal",
+                "order": 4,
+                "stops": {
+                    "1": [
+                        "28-1-1737"
+                    ],
+                    "0": [
+                        "28-0-419"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Franklin Park",
+                "branches": null,
+                "station": "frnpk",
+                "order": 5,
+                "stops": {
+                    "1": [
+                        "28-1-383"
+                    ],
+                    "0": [
+                        "28-0-415"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Grove Hall",
+                "branches": null,
+                "station": "ghall",
+                "order": 6,
+                "stops": {
+                    "1": [
+                        "28-1-386"
+                    ],
+                    "0": [
+                        "28-0-412"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Warren Street @ Boston Latin Academy",
+                "branches": null,
+                "station": "latac",
+                "order": 7,
+                "stops": {
+                    "1": [
+                        "28-1-390"
+                    ],
+                    "0": [
+                        "28-0-407"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Warren Street @ Walnut Avenue",
+                "branches": null,
+                "station": "warwl",
+                "order": 8,
+                "stops": {
+                    "1": [
+                        "28-1-396"
+                    ],
+                    "0": [
+                        "28-0-40001"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Nubian Station",
+                "branches": null,
+                "station": "nubn",
+                "order": 9,
+                "stops": {
+                    "1": [
+                        "28-1-64000"
+                    ],
+                    "0": [
+                        "28-0-64000"
+                    ]
+                }
+            },
+            {
+                "stop_name": "Roxbury Crossing Station (Bus)",
```

```suggestion
                "stop_name": "Roxbury Crossing Station",
```

(Looks like you already fixed this in the code, but didn't rerun the manifest generation fully for this one.)